cli.js

```diff
@@ -3,10 +3,12 @@
 /**
  * Parsing:
  *
- * $ debuguy parse <source_dir>
- * $ debuguy parse <source_dir> <destination_dir>
- * $ debuguy parse -r <source_dir>
- * $ debuguy parse -r <source_dir> <destination_dir>
+ * $ debuguy parse <source_dir> [options]
+ * $ debuguy parse <source_dir> <destination_dir> [options]
```

Comment:
You change the order of `options` here, but the option parsing logic is still the same.